### ISSUE TITLE
Feature request: ignore accents etc. in browser search 

### ISSUE THREAD

**Issue by thoral** [ankidroid/Anki-Android#5543]
It would be very useful for me if the search in the card browser could be improved in a way that accents etc. are ignored. I am searching very often a special term like, i.e. 'insólito', but if I omit the accent in the search input field (e. g. 'insolito') the result list is empty.

Of course this is nothing I expect, and I think also that everybody else would honour my feature request. For those who like to have a strict search there could be still an appropriate option.

I know that the original Anki program behaves the same, but there exists the possibility to loosen the strict search by means of a plugin (Ignore Accents in Browser Search).

Would be great if you could provide the same, e.g. with an appropriate option.

**Comment by mikehardy** [comment]
This is similar to, but not exactly a duplicate of #4821 

Any solution for that one will be the basis of a solution on this one.

There are lots of notes there for possible implementors if someone is interested.

**Comment by thoral** [comment]
Facilitating the search function is much smaller request than all the aspects which are described in #4821. Also, this is quite common in every search function used in webshops or search engines. Imagine you have to input accents in Google search or Wikipedia search forms to find something. All those letter 'modifications' which are mostly used to describe a different pronunciation are ignored in nearly every search function.

**Comment by mikehardy** [comment]
I believe I understand, the problem technically is as described in the linked issue though - I'm unsure the state of text normalization in our stack, whether it exists or not. If it exists both of these are unblocked, if it does not there's nothing we can do

**Comment by snoyes** [comment]
As a poor man's workaround, you can replace potentially accented characters with an underscore.
A search for `ins_lito` will match both (along with insilito and inselito and ins~lito).

**Comment by thoral** [comment]
Thanks for the hint. I did not know that SQL wildcards are allowed to input in the search form. Therefore also '%' is working as a wildcard character.

**Comment by thoral** [comment]
Perhaps it is possible in sqlite to use a NOACCENTS statement in the WHERE clause? See:
https://stackoverflow.com/questions/14009261/sqlite-accent-insensitive-search

**Comment by snoyes** [comment]
I imagine that would work, but you'd probably need to get https://github.com/dae/anki to do it first.

**Comment by thoral** [comment]
Just a remark: in Anki a new notation (nc:...) has been added in 2.1.24 to deal with such 'ignore accents' searches: 
https://betas.ankiweb.net/#/?id=changes-in-2124beta1

Personally, I don't like the nc: notation very much. It should be vice versa: the ones who need to search for the exact spelling should use a special syntax. All others should not care about accents and diacritic signs  etc. when they search anything. Common known online dictionaries like Pons search the same way: a word like 'começar' could be found if someone inputs only 'comecar'.  This is the way it should be in Anki, too, because it's the most easiest way for the user.

**Comment by mikehardy** [comment]
@thoral in general for any feature we will do it the same way Anki Desktop does it. I appreciate that you're having the dialog on the Anki Desktop site as it means the issue is under discussion. We won't deviate from them without a very good (usually platform-specific / something peculiar to Android and/or mobile) reason

**Comment by david-allison-1** [comment]
Bad bot
